src/groundlight/client.py

```diff
@@ -191,6 +250,8 @@ def submit_image_query(  # noqa: PLR0913 # pylint: disable=too-many-arguments
           Any binary format must be JPEG-encoded already.  Any pixel format will get
           converted to JPEG at high quality before sending to service.
         :param wait: How long to wait (in seconds) for a confident answer.
+        :param patience_time: How long Groundlight should work to generate a confident answer, even working beyond the specified wait time
```

Comment:
Should mention that this is optional, and that the detector’s default will be used if this is not specified.